### PR TITLE
changed dependency to avoid critical security issue.

### PR DESCRIPTION
**Pull request by MikkelTAndersen**
See issue here --> https://github.com/segmentio/analytics-java/issues/145

### CODE REVIEW

**Comment by MikkelTAndersen** [comment]
This is not enough the new dependency apparently requires a rewrite of the code.

File: analytics-core/src/main/java/com/segment/analytics/http/SegmentService.java [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-import retrofit.http.Body;
-import retrofit.http.POST;
+import retrofit2.Call;
+import retrofit2.http.Body;
+import retrofit2.http.POST;
 
 /** REST interface for the Segment API. */
 public interface SegmentService {
   @POST("/v1/import")
-  UploadResponse upload(@Body Batch batch);
```
Comment (f2prateek) [inline analytics-core/src/main/java/com/segment/analytics/http/SegmentService.java]: This is a breaking change.

Comment (f2prateek) [reply analytics-core/src/main/java/com/segment/analytics/http/SegmentService.java]: If there's no way to keep the declaration as `UploadResponse upload(@Body Batch batch);` with the built in functionality, we can use a custom call adapter.

**Comment by bbeaird** [comment]
analytics-java does not use XML as the serialization format, nor does it use the JAXB converter, so it's not impacted by this vulnerability. That said, we've got an internal ticket to resolve.

**Comment by nd4p90x** [comment]
PR #174 Resolved this issue

**Pull request closed without merging**